packages/salesforcedx-vscode-apex-replay-debugger/package.json

```diff
@@ -14,13 +14,16 @@
     "color": "#ECECEC",
     "theme": "light"
   },
+  "aiKey": "b093559c-0c91-43cb-ad24-67bb5a5cbc80",
```

Comment:
This aiKey would change when Nate gets us an official account right. The key right now is your personal experimental appinsights hub?

Reply:
Right, I'm waiting on Nate to get the official app insights key and using my dev account for now.

Reply:
We should have the official keys now? @ntotten

Reply:
@vazexqi this is something I missed removing, the official keys are on salesforcedx-vscode-code/package.json

Reply:
Yeah, we have had them for a while. `ec3632a4-df47-47a4-98dc-8134cacbaf7e` is for the prod account.